### PR TITLE
Fix the mismatch between issue command id vs trace command id

### PR DESCRIPTION
**Pull request by yalcinmelihyasin**
due to the initial commands

b/148923131

### CODE REVIEW

File: gapis/api/vulkan/find_issues.go [hunk 1]
```diff
@@ -295,7 +295,7 @@ func (t *findIssues) Flush(ctx context.Context, out transform.Writer) error {
 				return
 			}
 			// The debug report is issued for a trace command
-			issue.Command = api.CmdID(label - uint64(t.numInitialCmds))
+			issue.Command = api.CmdID(label)
```
Comment (pmuetschard) [inline gapis/api/vulkan/find_issues.go]: I don't think this change is a good idea. `issue.Command` should remain a valid command ID. To me it looks like this change just adds an offset of `t.numInitialCmds` here, just to remove it again when resolving the report.

File: gapis/resolve/report.go [hunk 2]
```diff
@@ -117,10 +118,13 @@ func (r *ReportResolvable) Resolve(ctx context.Context) (interface{}, error) {
 		}
 	}
 
+	InitialCommands, _, _ := initialcmds.InitialCommands(ctx, r.Path.Capture)
```
Comment (pmuetschard) [inline gapis/resolve/report.go]: lowercase variable names
I'm not a fan of this new dependency. Especially since it looks like it's just here to undo the change in `find_issues.go`.

File: gapis/api/vulkan/find_issues.go (hunk 1 above)
Comment (yalcinmelihyasin) [reply gapis/api/vulkan/find_issues.go]: Well, it is actually the other way around if you look at the rest of the code. 

The label here is a cmdID counted with the initial commands. We used to convert it by substracting the number of the initial commands. 

If we do the conversion here, we have no way to understand that if it's an issue coming find_issues.go with a cmdID without initial commands  or if it's an issue coming from externs.go with a cmdID including initial commands. We cannot resolve that at that point.

File: gapis/resolve/report.go (hunk 2 above)
Comment (yalcinmelihyasin) [reply gapis/resolve/report.go]: No, it is not what is happening here. 

This has to be done because the issues coming from externs.go counted with the initial commands.

We have two different way to create issues and they have been counted differently before, we have to unify them in a single place. And this seems to be the best place for it,

**Comment by yalcinmelihyasin** [comment]
This is updated and merged in new repo
